tests/auth_tests/test_forms.py

```diff
@@ -261,6 +261,21 @@ class Meta(UserCreationForm.Meta):
             ['The password is too similar to the first name.'],
         )
 
+    def test_autocomplete(self):
+        form = UserCreationForm()
+
+        tests = (
+            ('username', 'username'),
+            ('password1', 'new-password'),
+            ('password2', 'new-password')
```

Comment:
Missing another trailing comma here - please check all of these final elements in tuple, list and function arguments.